### ISSUE TITLE
BaseEmail class

### ISSUE THREAD

**Issue by benjaoming** [kbhff/eggplant#44]
There will be lots of different emails sent out and we need to unify these patterns at least into some kind of base class that provides a default template with configurable signature etc.

If sometime in 2016 someone wants to do HTML emails, they will be in a better position to do that because we obeyed DRY.

**Comment by valberg** [comment]
I suggest we look at using https://github.com/iconfinder/paloma for sending emails - it has exactly what you are talking about, that is a `Mail` class that does all the heavy lifting off sending mails.

**Comment by benjaoming** [comment]
Yeah, that's the same kind of class I'd normally copy-paste between my projects. Looks fine!
